sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1258,6 +1264,12 @@ object RapidsConf {
     .bytesConf(ByteUnit.BYTE)
     .createWithDefault(64 * 1024)
 
+  val SHUFFLE_MULTITHREADED_WRITER_THREADS =
+    conf("spark.rapids.shuffle.multiThreaded.writer.threads")
+      .doc("The number of threads to use for writing shuffle blocks per executor.")
+      .integerConf
+      .createWithDefault(20)
```

Comment:
This is going to need a similar followup issue to try to automatically tune this as we are working on for the multithreaded input readers.  20 may be a very poor choice on an executor with many configured cores.

Reply:
I think we can fold this into: https://github.com/NVIDIA/spark-rapids/issues/5039.

Reply:
I suggest to use `ForkJoinPool` class in this case. We can scale the number of threads dynamically using the API maximum pool size.
`getStealCount` can be used to evaluate how busy the threads are in the pool. if that returned value is too high, we can increase the number of workers.